pkg/agent/datamodel/types.go

```diff
@@ -101,11 +101,6 @@ const (
 	Ubuntu               Distro = "ubuntu"
 	Ubuntu1804           Distro = "ubuntu-18.04"
 	Ubuntu1804Gen2       Distro = "ubuntu-18.04-gen2"
-	RHEL                 Distro = "rhel"
-	CoreOS               Distro = "coreos"
-	AKS1604Deprecated    Distro = "aks"               // deprecated AKS 16.04 distro. Equivalent to aks-ubuntu-16.04.
-	AKS1804Deprecated    Distro = "aks-1804"          // deprecated AKS 18.04 distro. Equivalent to aks-ubuntu-18.04.
-	AKSDockerEngine      Distro = "aks-docker-engine" // deprecated docker-engine distro.
 	AKSUbuntu1604        Distro = "aks-ubuntu-16.04"
 	AKSUbuntu1804        Distro = "aks-ubuntu-18.04"
 	ACC1604              Distro = "acc-16.04"
```

Comment:
can you remove ACC1604 too, we don't use it in AKS

Reply:
Sure. ACC1604 removed.